cmd/ocm/delete/cmd.go

```diff
@@ -40,14 +40,67 @@ var args struct {
 }
 
 var Cmd = &cobra.Command{
-	Use:       "delete [flags] PATH",
+	Use:       "delete [flags] (PATH | RESOURCE_ALIAS RESOURCE_ID)",
```

Comment:
s/`(PATH | RESOURCE_ALIAS RESOURCE_ID)`/`(PATH|RESOURCE_ALIAS RESOURCE_ID)`/
I would remove the spaces to make it more transparent that it's PATH or RESOURCE_ALICAS.

Reply:
This format was taken from how `kubectl`/`oc` prints it. For example:

```
kubectl label --help
...
Usage:
  kubectl label [--overwrite] (-f FILENAME | TYPE NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--resource-version=version]
[options]
...
``